src/apyfixedarray_wrapper.cc

```diff
@@ -1231,7 +1231,38 @@ void bind_fixed_array(nb::module_& m)
                 An array filled with the specified value.
             )pbdoc"
         )
+        .def_static(
+            "arange",
+            &APyFixedArray::arange,
+            nb::arg("start"),
+            nb::arg("stop"),
+            nb::arg("step"),
+            nb::arg("int_bits") = nb::none(),
+            nb::arg("frac_bits") = nb::none(),
+            nb::arg("bits") = nb::none(),
+            R"pbdoc(
+            Initializes an array filled with the specified value.
```

Comment:
* `arange(stop)`: Values are generated within the half-open interval [0, stop) (in other words, the interval including start but excluding stop).
*  `arange(start, stop)`: Values are generated within the half-open interval [start, stop).
*  `arange(start, stop, step)`: Values are generated within the half-open interval [start, stop), with spacing between values given by step.